### PR TITLE
Python program implementation for Counting sort

### PR DESCRIPTION
**Pull request by sahebsunny**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

counting sort is an algorithm for sorting a collection of objects according to keys that are small integers.
Worst complexity: n+r
Average complexity: n+r
Space complexity: n+r

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#1772
I want to keep contributing to NeoAlgo.
Thank you.

### CODE REVIEW

File: Python/sort/Counting_Sort.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Implementation of the Counting Sort algorithm in Python.
+
+# Worst complexity: n+r
+# Average complexity: n+r
+# Space complexity: n+r
+# Stable: Yes
+# Class: Non-comparison sort
+# Data structure: Array
```
Comment (HarshCasper) [inline Python/sort/Counting_Sort.py:8]: We don't need to mention them

File: Python/sort/Counting_Sort.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
+# Implementation of the Counting Sort algorithm in Python.
+
+# Worst complexity: n+r
+# Average complexity: n+r
+# Space complexity: n+r
```
Comment (HarshCasper) [inline Python/sort/Counting_Sort.py:5]: Push them to the bottom of the Code

File: Python/sort/Counting_Sort.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+6                                                                                                                                  
+7                                                                                                                                  
+5                                                                                                                                  
+2                                                                                                                                  
+4                                                                                                                                  
+1                                                                                                                                  
+5                                                                                                                                  
+Sorted array in Ascending Order:                                                                                                   
+[1, 2, 3, 4, 5, 5, 6, 7] 
+"""
```
Comment (HarshCasper) [inline Python/sort/Counting_Sort.py:75]: Add a new line at the end of the Code

File: Python/sort/Counting_Sort.py [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+# Worst complexity: n+r
+# Average complexity: n+r
+# Space complexity: n+r
+# Stable: Yes
+# Class: Non-comparison sort
+# Data structure: Array
+
+
+def counting_Sort(arr):
```
Comment (HarshCasper) [inline Python/sort/Counting_Sort.py:11]: If possible add a Docstring for your Python function.

File: Python/sort/Counting_Sort.py [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+n = int(input("Enter length of array (n): "))
+
+lst = []
+print("Enter the Array (values has to be smaller than (n)): ")
+for _ in range(n):
+    lst.append(int(input()))
+
+counting_Sort(lst)
+print("Sorted array in Ascending Order: ")
+print(lst)
```
Comment (HarshCasper) [inline Python/sort/Counting_Sort.py:47]: Enclose this in a main function

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by atarax665** [comment]
@sahebsunny Closing this PR due to inactivity, comment if you are still working on this PR.

**Pull request closed without merging**